src/Controllers/NIPSWeb/confirm_central_upload.php

```diff
@@ -15,7 +15,8 @@
     $_REQUEST['title'],
     $_REQUEST['artist'],
     $_REQUEST['album'],
-    $_REQUEST['position']
+    $_REQUEST['position'],
+    $_REQUEST['explicit']
```

Comment:
something like:
isset(request) ? request === true : null;
would be my suggestion